pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -3,30 +3,30 @@
     "Microsoft.CodeAnalysis.CSharp": "1.3.0",
     "Microsoft.CodeAnalysis.VisualBasic": "1.3.0",
     "Microsoft.CSharp": "4.0.1",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0-beta-24415-01",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0-preview1-24508-06",
     "Microsoft.VisualBasic": "10.0.1",
     "NETStandard.Library": "1.6.1-beta-24415-01",
     "System.Buffers": "4.0.0",
     "System.Collections.Immutable": "1.2.0",
     "System.ComponentModel": "4.0.1",
     "System.ComponentModel.Annotations": "4.1.0",
     "System.Diagnostics.DiagnosticSource": "4.0.0",
-    "System.Diagnostics.Process": "4.1.1-beta-24415-01",
+    "System.Diagnostics.Process": "4.3.0-preview1-24508-08",
     "System.Dynamic.Runtime": "4.0.11",
```

Comment:
I only manually updated the ones that were required to get a clean build. I am relying on Maestro to perform the comprehensive update.